model_servers/llamacpp_python/base/Containerfile

```diff
@@ -1,5 +1,5 @@
 FROM registry.access.redhat.com/ubi9/python-311:1-62.1716478620
-ENV CHAT_FORMAT=llama-2
+ENV CHAT_FORMAT=
```

Comment:
I don't believe this works as you expect?  IE if I do

make CHAT_FORMAT=foobar, I don't think the CHAT_FORMAT variable in the podman build will get passed in.

Reply:
Understood. I can remove it, which just means we have no ability to set it at the build stage but we can default the option at the run stage.